src/main/java/service/util/ParserWriterImpl.java

```diff
@@ -0,0 +1,24 @@
+package service.util;
+
+import dao.StorageDao;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ParserWriterImpl implements ParserWriter {
+
+    static final String JAVA_CODE = "=";
```

Comment:
I don't get what is java_code and file_code, = isn't mentioned in  task description 🤔 
and fields like these should be private